docker/ci/dockerfiles/current/test.rockylinux8.systemd-base.x64.arm64.dockerfile

```diff
@@ -23,6 +23,11 @@ RUN dnf clean all && \
     dnf update -y && \
     dnf install -y which curl git gnupg2 tar net-tools procps-ng python3 python3-devel python3-pip zip unzip jq
 
+# Install gh cli
+RUN dnf clean all && dnf install -y 'dnf-command(config-manager)' && dnf config-manager --add-repo https://cli.github.com/packages/rpm/gh-cli.repo && \
+    dnf update -y && \
+    dnf install -y which curl git gnupg2 tar net-tools procps-ng python3 python3-devel python3-pip zip unzip jq gh
```

Comment:
```suggestion
    dnf install -y gh
```

Reply:
And one more question @gaiksaya , I see on of the images uses `yum` and the other uses `dnf` can I use the similar command to install gh. But both the installations are working

Reply:
Whatever is easier. @peterzhuamazon Any preference?

Reply:
> > Just install `gh` nothing else. Also can you add more details in description such as installing gh on docker images required for testing dashboards components? Fix the PR title too
> 
> The ubuntu image already have gh. We can actually add gh for all docker images if needed.
> 
> Thanks.

Added changes only to two images(rockylinux8) lacking gh used by OSD , I see there are two more docker files that do not have gh package, should I update them too?

Reply:
> 

yum is just a wrapper/symlink of dnf on RHEL8+, either is ok but certain commands relied on dnf, use dnf here is ok imo.

Reply:
> > > Just install `gh` nothing else. Also can you add more details in description such as installing gh on docker images required for testing dashboards components? Fix the PR title too
> > 
> > 
> > The ubuntu image already have gh. We can actually add gh for all docker images if needed.
> > Thanks.
> 
> Added changes only to two images(rockylinux8) lacking gh used by OSD , I see there are two more docker files that do not have gh package, should I update them too?

Please.